app/helpers/dates_helper.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module DatesHelper
+  def format_date(date)
```

Comment:
Is there a reason why we're not adding a custom format to `Date::DATE_FORMATS` and calling `Date#to_fs(:my_custom_format)` ? 
This works fine as is but it's not how most of our date formats are defined in [other projects](https://github.com/DFE-Digital/find-a-lost-trn/blob/main/config/initializers/date_formats.rb).

Reply:
No particular reason, I didn't know that existed! I'll amend it to use that.